### ISSUE TITLE
kinto: documentation fails to build as of 2.1.2

### ISSUE THREAD

**Issue by glasserc** [sphinx-doc/sphinx#6544]
An automatic dependency bump (https://github.com/Kinto/kinto/pull/2182) of Sphinx from 2.1.1 to 2.1.2 is failing. We receive the error:

```
Warning, treated as error:
/home/travis/build/Kinto/kinto/kinto/core/openapi.py:docstring of kinto.core.openapi.OpenAPI.generate::py:class reference target not found: dict
```

`OpenAPI.generate` doesn't actually have a docstring; it overrides https://github.com/Cornices/cornice.ext.swagger/blob/master/cornice_swagger/swagger.py#L398. This has `:rtype: dict`. We have other functions with this `:rtype: dict` field in our project and we have `("py:obj", "dict")` on our `nitpick_ignore` list (since Sphinx 2.1.1; https://github.com/Kinto/kinto/pull/2171).

```
$ git clone https://github.com/Kinto/kinto
$ cd kinto
$ make docs
```

**Expected behavior**
I didn't see anything in the changelog that would explain why this would fail.

**Your project**
https://github.com/Kinto/kinto/
https://travis-ci.org/Kinto/kinto/jobs/548162675 shows a failing build.

**Environment info**
- OS: Linux (container on TravisCI)
- Python version: 3.6.3
- Sphinx version: 2.1.2
- Sphinx extensions:  `sphinx.ext.autodoc`, `sphinxcontrib.httpdomain`, `sphinx.ext.extlinks`, `sphinx.ext.intersphinx`; the `sphinx_rtd_theme`
- Extra tools: Readthedocs, Travis

**Comment by tk0miya** [comment]
Sorry for inconvenience. The behavior of 2.1.1 was not expected. And it was fixed at 2.1.2 (refs: #6478). So please update your `nitpick_ignore` setting to `("py:class", "dict")`.

**Comment by glasserc** [comment]
Thank you very much!
